services/todo-ui/src/browser/components/TodoListActions.tsx

```diff
@@ -7,7 +7,7 @@ import { Box, Typography, Button } from '@mui/material';
 import React from 'react';
 import { pluralize } from '../utils/utils';
 import { Filter } from '../pages/todoweb/TodowebPage';
-import { TasksClient } from '../clients/TasksClient';
+import { TasksClient } from '../.generated/clients/TasksClient';
```

Comment:
Should we use the "generated:" path alias in all our imports to demonstrate that is the recommended way?

Reply:
In react that's more complicated - since we've got several .generated folders